torch/csrc/api/src/nn/modules/batchnorm.cpp

```diff
@@ -72,5 +78,75 @@ Tensor BatchNormImpl::pure_forward(
       torch::cuda::cudnn_is_available());
 }
 
+template <size_t D, typename Derived>
+BatchNormImplBase<D, Derived>::BatchNormImplBase(const BatchNormOptionsv2<D>& options_)
+    : options(options_) {
+  if (options.affine()) {
+    weight = this->register_parameter("weight", torch::ones({options.num_features()}));
+    bias = this->register_parameter("bias", torch::zeros({options.num_features()}));
+  }
+  if (options.track_running_stats()) {
+    running_mean = this->register_buffer("running_mean", torch::zeros({options.num_features()}));
+    running_var = this->register_buffer("running_var", torch::ones({options.num_features()}));
+    num_batches_tracked = this->register_buffer("num_batches_tracked", torch::tensor(0, torch::dtype(torch::kLong)));
```

Comment:
I think we should keep `num_batches_tracked` as `momentum` can still be null in the C++ version :D

Reply:
Thanks. I added the codes if `options.track_running_stats()` is `false`.
Do these codes match your intention?